.eslintignore

```diff
@@ -1,4 +0,0 @@
-node_modules
-electron/dist
-electron/node_modules
-electron/renderer/dist
```

Comment:
Are you sure this is not needed?!

Try running the tests.. if it takes 3 hours, revert this file.